### PR TITLE
Contact us

### PR DESCRIPTION
**Pull request by ThaiBangHOANG**
I have just modified the ContactUs page. Would you take a look and valid for me, please?
Thank you,

Thai Bang

### CODE REVIEW

**Review by Hacene010 — DISMISSED** [review]

File: src/components/ContactUs.jsx [hunk 1]
```diff
@@ -1,30 +1,54 @@
+import React from 'react';
```
Comment (LoicBrassart) [inline src/components/ContactUs.jsx]: That's a useless import, drop it

File: src/components/ContactUs.jsx [hunk 2]
```diff
@@ -1,30 +1,54 @@
...
+            <input type="number" id="phone" placeholder="Your phone number" />
+          </label>
+          <p className="form textArea">How do we get better?</p>
+          <label htmlFor="message" className="form">
+            <textarea id="message" placeholder="Your message here">
+              :)
+            </textarea>
+          </label>
+          <br />
+          <a href="http://www.google.com">
```
Comment (LoicBrassart) [inline src/components/ContactUs.jsx]: This link is useless, we will see how to add behavior to a button, you can delete it

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A bit of cleanup to do, but it's mostly fine

File: src/components/ContactUs.jsx (hunk 1 above)
Comment (ThaiBangHOANG) [reply src/components/ContactUs.jsx]: Thank you. I have just deleted it.

File: src/components/ContactUs.jsx (hunk 2 above)
Comment (ThaiBangHOANG) [reply src/components/ContactUs.jsx]: Yes I have just deleted it also

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evils

**Pull request merged**